### PR TITLE
[BD-46] docs: new component examples and editable live code blocks

### PR DESCRIPTION
**Pull request by PKulkoRaccoonGang**
## Description

- [x] Ensure rendered components for each live code blocks are rendered on a light background (e.g., gray-200, light-200) to more clearly delineate where the code examples are on the page.
- [x]  Ensure collapsible for showing the live code editor is rendered as the large card style (i.e., styling="card-lg").
- [x] Ensure an icon (default Collapsible icon is likely fine) is rendered as an affordance to indicate that the live code block may be collapsed/expanded.
- [x]  Discovery: make a recommendation for implementing a "Copy" to clipboard (icon)button for live code blocks to allow engineers/consumers to more easily copy/paste from the component code snippets on the docs site. If implemented, ensure it gets a Segment event when clicked where the event name follows the same convention as other Segment event names throughout the docs site.

Issue: https://github.com/openedx/paragon/issues/1981

### Deploy Preview

[Paragon](https://deploy-preview-2194--paragon-openedx.netlify.app/components/actionrow/)

## Merge Checklist

* [ ] If your update includes visual changes, have they been reviewed by a designer? Send them a link to the Netlify deploy preview, if applicable.
* [x] Does your change adhere to the documented [style conventions](https://github.com/openedx/paragon/blob/master/docs/decisions/0012-css-styling-conventions)?
* [x] Do any prop types have missing descriptions in the Props API tables in the documentation site (check deploy preview)?
* [x] Were your changes tested using all available themes (see theme switcher in the header of the deploy preview, under the "Settings" icon)?
* [ ] Were your changes tested in the `example` app?
* [ ] Is there adequate test coverage for your changes?
* [ ] Consider whether this change needs to reviewed/QA'ed for accessibility (a11y). If so, please add `wittjeff` and `adamstankiewicz` as reviewers on this PR.

## Post-merge Checklist

* [ ] Verify your changes were released to [NPM](https://www.npmjs.com/package/@edx/paragon) at the expected version.
* [ ] If you'd like, [share](https://github.com/openedx/paragon/discussions/new?category=show-and-tell) your contribution in [#show-and-tell](https://github.com/openedx/paragon/discussions/categories/show-and-tell).
* [ ] 🎉 🙌 Celebrate! Thanks for your contribution.

### CODE REVIEW

**Comment by openedx-webhooks** [comment]
Thanks for the pull request, @PKulkoRaccoonGang!


When this pull request is ready, tag your edX technical lead.

File: www/src/components/CodeBlock.tsx [hunk 1]
```diff
@@ -23,30 +23,40 @@ import * as ParagonIcons from '~paragon-icons';
 import MiyazakiCard from './exampleComponents/MiyazakiCard';
 import HipsterIpsum from './exampleComponents/HipsterIpsum';
 import ExamplePropsForm from './exampleComponents/ExamplePropsForm';
+import { ContentCopy } from '../../../icons';
```
Comment (viktorrusakov) [inline www/src/components/CodeBlock.tsx:26]: nit: we have `~paragon-icons` alias for such imports

Comment (PKulkoRaccoonGang) [reply www/src/components/CodeBlock.tsx]: done

**Review by adamstankiewicz — APPROVED** [review]
This LGTM to run with for now! We can always make additional design tweaks later on, if needed. This is definitely an improvement in the usability of the code blocks, though! 😄

**Comment by openedx-webhooks** [comment]
@PKulkoRaccoonGang 🎉 Your pull request was merged!  Please take a moment  to [answer a two question survey](https://docs.google.com/forms/d/e/1FAIpQLSceJOyGJ6JOzfy6lyR3T7EW_71OWUnNQXp68Fymsk3MkNoSDg/viewform?usp=pp_url&entry.1671973413=openedx/paragon&entry.867055334=https://github.com/openedx/paragon/pull/2194&entry.1484655318=https://github.com/PKulkoRaccoonGang&entry.752974735=2023-04-14+15:20&entry.1917517419=2023-04-21+11:39&entry.2133058324=Yes) so we can improve your experience in the future.

**Pull request merged**